.github/workflows/lint-extras.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: lint-extras
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  lint-extras:
+    name: lint-extras
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: lint-extras
```

Comment:
there is no linter action here, so the job only checksout the git repository, nothing more

Maybe you want to use super-linter like is done in https://github.com/csi-addons/spec/blob/main/.github/workflows/linter.yaml ?

Reply:
Right Niels, Thank you! 
Updated the PR.